test/jit/test_save_load_for_op_version.py

```diff
@@ -425,3 +421,35 @@ def _helper(m, fn):
                 self.assertEqual(mr, hr)
 
         _helper(v3_mobile_module, current_mobile_module)
+
+    def test_versioned_linspace(self):
+        scripted_module = torch.jit.load(
+            pytorch_test_dir + "/jit/fixtures/test_versioned_linspace_v7.ptl")
+
+        buffer = io.BytesIO(scripted_module._save_to_buffer_for_lite_interpreter())
+        buffer.seek(0)
+        v7_mobile_module = _load_for_lite_interpreter(buffer)
+
+        sample_inputs = ((3, 10), (-10, 10), (4.0, 6.0), (3 + 4j, 4 + 5j))
+        for (a, b) in sample_inputs:
+            (output_with_step, output_without_step) = v7_mobile_module(a, b)
+            # when no step is given, should have used 100
+            self.assertTrue(output_without_step.size(dim=0) == 100)
+            self.assertTrue(output_with_step.size(dim=0) == 5)
```

Comment:
nit, maybe tedious, still better to use variable here `self.assertTrue(output_without_step.size(dim=0) == expect_output)`